docs/src/components/footer/nci-big.jsx

```diff
@@ -24,14 +24,17 @@ export const NCIBigFooter = () => {
 											} `}>
 											<button
 												onClick={() => setActiveItem(activeItem === 1 ? 0 : 1)}
-												className="usa-button usa-button--unstyled">
-												<h4 className="usa-footer__primary-link">
-													Primary link 1
+												className="usa-button usa-button--unstyled usa-footer__nci-accordion-button">
+												<h4 className="usa-footer__primary-link usa-footer__nci-primary-link--accordion-header">
```

Comment:
I don't think headings are allowed inside of buttons.  I'm surprised linting didn't complain.

Reply:
do we even need a heading here? styles are all in a class, not attached to any `h*`
could use a span or div instead

Reply:
This has been fixed